src/TestUtils/src/Microsoft.Maui.IntegrationTests/AppleTemplateTests.cs

```diff
@@ -49,6 +49,7 @@ public void RunOniOS(string id, string config, string framework, string runtimeI
 				buildProps.Add("PublishAotUsingRuntimePack=true"); // TODO: This parameter will become obsolete https://github.com/dotnet/runtime/issues/87060
 				buildProps.Add("_IsPublishing=true"); // using dotnet build with -p:_IsPublishing=true enables targeting simulators
 				buildProps.Add($"RuntimeIdentifier={runtimeIdentifier}");
+				buildProps.Add("IlcTreatWarningsAsErrors=false"); // TODO: Remove this once all warnings are fixed https://github.com/dotnet/maui/issues/19397
```

Comment:
Thanks for fixing this! 
I will create a separate PR for all `IlcTreatWarningsAsErrors=false` in net8.0 branch as a follow-up, as we might backport this feature to net8.0 on the dotnet/runtime side.